features/miniAlert.js

```diff
@@ -0,0 +1,23 @@
+import { settings } from "../settings";
+

```

Comment:
look at the feature structure and implement it so it properly works, add it to the loader